tests/plotting/test_plotting.py

```diff
@@ -1401,27 +1400,92 @@ def test_read_texture_from_numpy():
     plotter.show()
 
 
-def test_plot_rgb():
-    """Test adding a texture to a plot"""
-    cube = pv.Cube()
-    cube.clear_data()
-    x_face_color = (255, 0, 0)
-    y_face_color = (0, 255, 0)
-    z_face_color = (0, 0, 255)
-    face_colors = np.array(
-        [
-            x_face_color,
-            x_face_color,
-            y_face_color,
-            y_face_color,
-            z_face_color,
-            z_face_color,
-        ],
-        dtype=np.uint8,
-    )
-    cube.cell_data['face_colors'] = face_colors
+def _make_rgb_dataset(dtype: str, return_composite: bool, scalars: str):
+    def _dtype_convert_func(dtype):
+        # Convert color to the specified dtype
+        if dtype == 'float':
+
+            def as_dtype(color: tuple[float, float, float]):
+                return pv.Color(color).float_rgb
+        elif dtype == 'int':
+
+            def as_dtype(color: tuple[float, float, float]):
+                return pv.Color(color).int_rgb
+        elif dtype == 'uint8':
+
+            def as_dtype(color: tuple[float, float, float]):
+                return np.array(pv.Color(color).int_rgb, dtype=np.uint8)
+        else:
+            raise NotImplementedError
+
+        return as_dtype
```

Comment:
```suggestion
        # Convert color to the specified dtype
        if dtype == 'float':
            def as_dtype(color: tuple[float, float, float]):
                return pv.Color(color).float_rgb
        elif dtype == 'int':
            def as_dtype(color: tuple[float, float, float]):
                return pv.Color(color).int_rgb
        elif dtype == 'uint8':
            def as_dtype(color: tuple[float, float, float]):
                return np.array(pv.Color(color).int_rgb, dtype=np.uint8)
        else:
            raise NotImplementedError
        return as_dtype
```

Reply:
I'm fine with this, but the formatters (black? Maybe ruff?) want to see a blank line before function definitions, so this will probably be reverted with ci.auto-fix. We could try using lambda functions instead, though ruff may complain about that too and suggest using a function def instead.

Reply:
I'm fine if formatter say so. Let's keep as is :)